src/dynamic_placeholder_text_util.c

```diff
@@ -7,17 +7,11 @@ static EWRAM_DATA const u8 *sStringPointers[8] = {};
 
 void DynamicPlaceholderTextUtil_Reset(void)
 {
-    const u8 **ptr;
-    u8 *fillval;
-    const u8 **ptr2;
-
-    ptr = sStringPointers;
-    fillval = NULL;
-    ptr2 = ptr + (ARRAY_COUNT(sStringPointers) - 1);
-    do
+    s32 i;
+    for (i = 0; i < (s32)ARRAY_COUNT(sStringPointers); i++)
     {
```

Comment:
The curly brackets here can be removed. The project doesn't lay out a scope for `for` statements that execute 1 single operation.